### PR TITLE
Section/header

### PR DESCRIPTION
**Pull request by whiteblackswan**

### CODE REVIEW

**Review by Dmitruk89 — CHANGES_REQUESTED** [review]

File: sass/_general.sass [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.nav-links li a
+	font-weight: 400
+	font-size: 20px
+	line-height: 27px
+	text-align: center
+	display: block
+	padding: 20px
+	color: $main-white
+
+.nav-links li a:hover
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply sass/_general.sass:50]: I fixed everything

File: sass/_home.sass [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+				display: inline-block
+				font-weight: 700
+				font-size: 20px
+				line-height: 27px
+				text-align: center
+				color: $maim-black
+				img
+					margin-left: 5px
+		.button:hover
+			background-color: yellow
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply sass/_home.sass:45]: I fixed everything

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// activate our mobile menu button
+
+const menuButton = document.getElementsByClassName('menu-button')[0]
+
+const navLinks = document.getElementsByClassName('nav-links')[0]
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply js/script.js:5]: I fixed everything

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+// create an even listener
+menuButton.addEventListener('click', () => {
+	navLinks.classList.toggle('active')
+	menuButton.classList.toggle('active')
+})
+
+
+
+menuButton.forEach(n => n.addEventListener("click", closeMenu));
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply js/script.js:15]: I fixed everything

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+function closeMenu() {
+	menuButton.classList.remove("active");
+	navMenu.classList.remove("active");
+}
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply js/script.js:20]: I fixed everything

File: sass/_general.sass [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	text-align: center
+	color: $main-white
+.nav-links
+	margin-inline-end: 2px
+
+li
+	list-style: none
+a
+	text-decoration: none
+	font-size: 16px
```
[the remark this answers is not in the document]
Comment (whiteblackswan) [reply sass/_general.sass:25]: I fixed everything

**Review by Dmitruk89 — APPROVED** [review]

**Pull request merged**